vtm.js

```diff
@@ -261,12 +383,41 @@ app.component('stat-section', {
         :key="list.id"
         :categ="list"
         :scale="stats.resource.length - 1"
-        @stat-category-change="emitAllowedChange($event)"
-      >
+        @stat-category-change="emitAllowedChange($event)">
       </stat-category>      
     </div>`
 });
 
+/** Displays entire section of skills
+ * displays select for distribution of skills
+ * receives events from child component and check if change is possible in resources
+ * if changes are possible, emit event to top component to make changes, otherwise don't
+ */
+app.component('skill-section', {
+  mixins:[statSectionMixin,attributesAndSkillsMixin],
+  props:['distributions'],
+  template: `
+    <div class="statSection">
+      <h2>{{stats.id}}</h2>
+      <select v-model="stats.resource">
+        <option disabled value="">Select a distribution</option>
+        <option 
+          v-for="distrib in distributions"
+          :key = "distrib.id"
+          :value="distrib.resource">
+          {{distrib.id}}
+        </option>
+      </select>
+      <div class="resourceCount">{{allocatedResources}} | {{stats.resource}}</div>
+      <stat-category
+        v-for="list in stats.data"
+        :key="list.id"
+        :categ="list"
```

Comment:
category?

Reply:
to si mohol urobit aj sam

Reply:
no hej, ale potom by si urcite nevedel :-) ze zkratky podla vue3 style guide niesu dobre pre citelnost :-))

Reply:
to ja viem... ale to categ tam je uz hodne dlho, tak mesiac alebo nieco take a nevadilo ti to pred tym

Reply:
chces sa dohadovat, alebo to opravis? :-)

vsimol som si ze to tam teraz pridavas, tak som ti to napisal